BackEnd/administration/serializers.py

```diff
@@ -51,15 +51,39 @@ def create(self, validated_data):
 
 
 class AdminUserListSerializer(serializers.ModelSerializer):
+    company_name = serializers.SerializerMethodField()
+    registration_date = serializers.SerializerMethodField()
+    status = serializers.SerializerMethodField()
+
     class Meta:
         model = CustomUser
         fields = (
             "id",
             "email",
             "name",
             "surname",
+            "status",
+            "company_name",
+            "registration_date",
         )
 
+    def get_company_name(self, obj) -> str:
+        return obj.profile.name if hasattr(obj, "profile") else "No profile"
```

Comment:
Wouldn't it be better to return None instead of "No profile" if the user doesn't have a profile?

Reply:
Done